### PR TITLE
Boundary Conditions are Validated

### PR DESCRIPTION
**Pull request by CodyFagley**
Validate button stays inactive until a load/anchor are *both* applied.   

Messages update accordingly, but will be ultimately changed to match Teton Mockups.

### CODE REVIEW

File: SmartSlicePlugin/SmartSliceSelectTool/SmartSliceSelectTool.py [hunk 1]
```diff
@@ -137,18 +137,37 @@ def _onSelectedFaceChanged(self):
             Logger.log("d", "loaded_faces: {}".format(loaded_faces))
             Logger.log("d", "anchored_faces: {}".format(anchored_faces))
             
-            cloud_connector = PluginRegistry.getInstance().getPluginObject("SmartSliceExtension").cloud
             if self._selection_mode is SelectionMode.AnchorMode:
+                print ("\n\nANCHOR APPLIED RIGHT HERE\n\n")
```
Comment (bradymadams) [inline SmartSlicePlugin/SmartSliceSelectTool/SmartSliceSelectTool.py]: Is this print statement (and the one below) for debugging? Or do we want to keep it?

File: SmartSlicePlugin/SmartSliceExtension/SmartSliceCloudProxy.py [hunk 2]
```diff
@@ -18,13 +18,14 @@
 class SmartSliceCloudStatus:
```
Comment (bradymadams) [inline SmartSlicePlugin/SmartSliceExtension/SmartSliceCloudProxy.py]: We should inherit from `enum.Enum`

**Comment by bradymadams** [comment]
Accidentally deleted the source branch which closed the PR. Branch has been restored and PR re-opened.

File: SmartSlicePlugin/SmartSliceSelectTool/SmartSliceSelectTool.py (hunk 1 above)
Comment (CodyFagley) [reply SmartSlicePlugin/SmartSliceSelectTool/SmartSliceSelectTool.py]: This has been removed as part of the nextmerge request, I believe.

File: SmartSlicePlugin/SmartSliceExtension/SmartSliceCloudProxy.py (hunk 2 above)
Comment (CodyFagley) [reply SmartSlicePlugin/SmartSliceExtension/SmartSliceCloudProxy.py]: Confirmed; I just read up on this.  I will switch our enumerations to enum.Enum as part of best practice.

**Pull request merged**